app/move/steps/create.js

```diff
@@ -152,6 +152,10 @@ module.exports = {
     pageTitle: 'moves::steps.document.heading',
     fields: ['documents'],
   },
+  '/date-range': {
```

Comment:
This might be better called after what is happening on the step `/move-date` for consistency?

All our other step URLs attempt to explain what we're asking so we have a sensible URL structure.

Reply:
Just noticed the position will also need to move. It should come over the `/move-details` step. The other of the properties in the steps object determine the order in the form wizard so its important they sit in the right place.